Gandalan.IDAS.WebApi.Client/BusinessRoutinen/SerienMaterialBerechnenWebRoutinen.cs

```diff
@@ -29,9 +29,9 @@ public async Task<List<MaterialbedarfDTO>> GetMaterialAsync(Guid serieGuid)
             => await GetAsync<List<MaterialbedarfDTO>>("SerieMaterialbedarfBerechnen?serieGuid=" + serieGuid.ToString());
 
         public async Task<List<MaterialbedarfDTO>> GetOffenerMaterialBedarfAsync(Guid serieGuid)
-            => await GetAsync<List<MaterialbedarfDTO>>("SerieOffenerMaterialbedarf?serieGuid=" + serieGuid.ToString());
+            => await GetAsync<List<MaterialbedarfDTO>>("SerieOffenerMaterialbedarf?serieGuid=" + serieGuid.ToString()/*, version: "2"*/);
         public async Task<List<MaterialbedarfDTO>> GetOffenerMaterialBedarfv1Async(Guid serieGuid)
```

Comment:
Do it vice versa: keep the original method signature to call v1, and use an explicit v2 in the method name for v2 API call